xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -67,7 +67,7 @@ extern "C"
     };
 
     /// set of supported capabilities
-    uint32_t m_mask;
+    uint32_t m_mask = 0;
```

Comment:
Here and on other places not match a "C" style for there.
They are thought to use also (in future) as pure "C".

Also some other parts (already before, not from you) are not "C" conform and need improved.

Is already set to 0 here: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp#L58

Better to extend this be make a `m_caps = {{0}};` to make sure it stays 0 if another part becomes added

Reply:
Ok, done as you suggested.